src/cart.ts

```diff
@@ -548,6 +549,43 @@ class Cart extends Node<Graph.Cart> {
 
     return this.refresh(response.cartAddToWaitlist.cart);
   }
+
+  async sendClientAuthorizationCodeViaSms(mobilePhone: string): Promise<string> {
```

Comment:
I'd love to see some docs here just as for the other functions (once we have passwordless auth described in the dev portal, it would be nice to link that as well)